test/helpers/role_organisations_helper_test.rb

```diff
@@ -25,5 +25,14 @@ class RoleOrganisationsHelperTest < ActionView::TestCase
       assert user_organisation_option[:selected]
       assert_not other_organisation_option[:selected]
     end
+
+    should "sort by organisation name alphabetically" do
+      options = options_for_your_organisation_select(@user)
+
+      assert_equal(
+        [@other_organisation.name_with_abbreviation, @user_organisation.name_with_abbreviation],
```

Comment:
Subjective: It's not immediately obvious to me how this test is testing the ordering. This is because: (a) the use of the instance variables means I have to look up at the setup to see what they are; and (b) because even then neither organisation has its `name` set explicitly - I assume it's relying on the Organisation factory naming sequence. Of the two points, I think (b) is the one I care about more.

I think this (and other tests) in this test case would be easier to follow if the organisation names were set explicitly and those string literals were repeated in the assertions even at the cost of a tiny bit of duplication.

Reply:
Addressed in [edac91f](https://github.com/alphagov/signon/pull/2417/commits/edac91fefe6c6d3baab9c1e8115ed062f74fbe29)